packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -159,18 +161,31 @@ async function createExtensions(
   output = true,
 ) {
   const result: {[localIdentifier: string]: RemoteSource} = {}
+  let counter = 0
   for (const extension of extensions) {
-    // Create one at a time to avoid API rate limiting issues.
-    // eslint-disable-next-line no-await-in-loop
-    const registration = await createExtension(
-      appId,
-      extension.graphQLType,
-      extension.handle,
-      developerPlatformClient,
-      extension.contextValue,
-    )
-    if (output) outputCompleted(`Created extension ${extension.handle}.`)
-    result[extension.localIdentifier] = registration
+    counter++
+    if (developerPlatformClient.supportsAtomicDeployments) {
```

Comment:
This shouldn't actually be invoked for configuration extensions, but we'll eventually hit this when we support other extension types.